app/lib/backend/auth.dart

```diff
@@ -110,7 +110,9 @@ Future<UserCredential?> signInWithGoogle() async {
   try {
     debugPrint('Signing in with Google');
     // Trigger the authentication flow
-    final GoogleSignInAccount? googleUser = await GoogleSignIn(scopes: ['profile', 'email']).signIn();
+    final GoogleSignInAccount? googleUser = await GoogleSignIn(scopes: ['profile', 'email'],
+      clientId: "1031333818730-l9meebge7tpc0qmquvq6t5qpoe09o1ra.apps.googleusercontent.com",
+    ).signIn();
```

Comment:
its already in `app/setup/prebuilt/google-services.json` - why do we need it here? this hardcode is not prod ready.